nntrainer.ini.in

```diff
@@ -6,3 +6,10 @@
 [plugins]
 # path to search for layers
 layer=@PLUGIN_INSTALL_PREFIX@/layers
+
+[swap]
+# enable memory swap feature
+memory_swap = false
+
+# path to save swap file
+memory_swap_path = ./
```

Comment:
I wonder that if the current directory of swap file is intended. I think it might be configured depending on platform where we can have write permission using alias like &PLUGIN_INSTALL_PREFIX&. BTW Please check the new line here.

Reply:
I understood. MEMORY_SWAP and MEMORY_SWAP_PATH envs are added. And, default MEMORY_SWAP_PATH is set as:
android:  /data/local/tmp
other: /tmp

Thanks for your review.